pkg/driver/node_server.go

```diff
@@ -256,6 +256,27 @@ func (driver *Driver) nodeStageVolume(
 		return nil // volume already staged, do nothing and return here
 	}
 
+	if volumeContext[volEncryptionSecretNameKey] != "" {
```

Comment:
Change to boolean as per above request.

Reply:
This will still be needed to validate the user supplied value.